docs/logs/configuring.asciidoc

```diff
@@ -3,95 +3,48 @@
 
 :ecs-link: {ecs-ref}[Elastic Common Schema (ECS)]
 
-== Configuring the Logs UI
+== Configuring the Logs data
 
-The `filebeat-*` index pattern is used to query data by default. If your logs
-are located in a different set of indices, use a different timestamp field, or
-contain parsed fields which you want to expose as individual columns, you can
-adjust the source configuration via the user interface or the {kib}
-configuration file.
+The default source configuration for logs is specified in the {kibana-ref}/logs-ui-settings-kb.html[Logs app settings] in the {kibana-ref}/settings.html[Kibana configuration file].
+The default configuration uses the `filebeat-*` index pattern to query the data.
+The default configuration also defines field settings for things like timestamps and container names, and the default columns to show in the logs pane.
 
-NOTE: Logs and Infrastructure share a common data source definition in
-each space. Changes in one of them can influence the data displayed in the
-other.
+If your logs have custom index patterns, or use non-default field settings, or contain parsed fields which you want to expose as individual columns, you can override the default settings.
+Click *Configuration* to change the settings.
+This opens the *Configure source* fly-out dialog.
 
-[float]
-=== Configure source
-
-*Configure source* can be accessed via
-image:logs/images/logs-configure-source-gear-icon.png[Configure source icon]
-in the toolbar.
-
-[role="screenshot"]
-image::logs/images/logs-configure-source.png[Configure Logs UI source button in Kibana]
+NOTE: These settings are shared with metrics. Changes you make here may also affect the settings used by the *Metrics* app.
 
-This opens the source configuration fly-out dialog with multiple tabs, where
-you can inspect and adjust various index settings and log column configuration.
-
-TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration
-changes performed via *Configure source* are specific to that space. You can
-therefore easily make different subsets of the data available by creating
-multiple spaces with different data source configurations.
-
-[float]
-[[logs-read-only-access]]
```

Comment:
Just noticed that this ID was removed. It needs to be added to redirects.asciidoc.

Reply:
Aha, that's the Logs equivalent of the Infrastructure one I removed. Such a tiny, specific thing I cannot see it ever being needed, but I'll add it as before. Good spot!